acceptancetests/jujupy/client.py

```diff
@@ -1457,24 +1457,19 @@ def _maas_spaces_enabled():
     def _get_substrate_constraints(self):
         if self.env.joyent:
             # Only accept kvm packages by requiring >1 cpu core, see lp:1446264
-            return 'mem=2G cpu-cores=1'
+            return 'cores=1'
```

Comment:
is it `cpu-cores` or `cores`?

Reply:
should be cores.  i saw a warning during bootstrap of joyent: 
Warning: constraint "cpu-cores" is deprecated in favor of "cores".